naga/src/valid/handles.rs

```diff
@@ -1,17 +1,17 @@
 //! Implementation of `Validator::validate_module_handles`.
 
+use super::ValidationError;
+use crate::non_max_u32::NonMaxU32;
 use crate::{
     arena::{BadHandle, BadRangeError},
     diagnostic_filter::DiagnosticFilterNode,
     Handle,
 };
-
-use crate::non_max_u32::NonMaxU32;
 use crate::{Arena, UniqueArena};
+use core::{convert::TryInto, hash::Hash};
 
-use super::ValidationError;
-
-use std::{convert::TryInto, hash::Hash};
+#[cfg(test)]
+use alloc::string::ToString;
```

Comment:
I know we don't follow this super strictly, but we should try to follow rustfmt's [StdExternalCrate](https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#StdExternalCrate%5C%3A) ordering for imports. You don't need to fix things you didn't touch, but when changing them, we should try to standardize them.